### DISCUSSION TITLE
please add the possibility to place a link into the description or somewhere.

### DISCUSSION THREAD

**Discussion by swinxx** [jordan-dalby/ByteStash#54] [Ideas]
i am new in the self hosted playground, and i am blown away.. so many things to learn. i also have to learn yaml. and i think your project is nearly perfect for me to place code snippets from different sources. and thats what i really would like to reference to. so if bytestash could understand web links in the description or somewhere else, this would be perfect for me. i need some cross links, cause its soo much info right at the moment. please consider adding this in a future release. thank you for your hard work!! best wishes.

**Comment by jordan-dalby** [comment]
Hey, thanks for the idea!
#55
